src/docs/testing/errors.md

```diff
@@ -43,6 +43,10 @@ void main() {
 
 // rest of `flutter create` code...
 ```
+{{site.alert.note}} 
+  [kReleaseMode](https://api.flutter.dev/flutter/foundation/kReleaseMode-constant.html) is a 
+  top-level constant to check if the application was compiled in release mode.
+{{site.alert.end}}
```

Comment:
This is a helpful note! But can you tweak it a bit?

```
{{site.alert.note}}
  Use the top-level [`kReleaseMode`][] constant to determine whether the
  app was compiled in release mode.
{{site.alert.end}}
```

And at the bottom of the file:

```
[`kReleaseMode`]:  {{site.api}}/flutter/foundation/kReleaseMode-constant.html
```

Reply:
I have made changes as per your instruction.